src/stores/models/Oracle.js

```diff
@@ -123,5 +125,11 @@ export default class Oracle {
     this.endTime = this.phase === RESULT_SETTING ? oracle.resultSetEndTime : oracle.endTime;
 
     this.options = oracle.options.map((option, i) => new Option(option, i, this, app));
+
+    this.localizedInvalid = {
```

Comment:
lets make a convenience getter method for getting the array of options. something like:
```
getOptions = (locale) => {
  const localizedInvalid = {
    en: 'Invalid',
    zh: '无效',
    ko: '무효의',
  }
  return _.map(this.options, (option) => {
    if (option === 'Invalid') {
      return localizedInvalid[locale.slice(0, 2)];
    }
    return option;
  });
}
```

haven't tested the above code, but i think its correct.

then just use this method anytime you get the options array.

Reply:
I made a parse function inside localizedInvalid instead of this mapping since what calls it is already inside a mapping.